### PR TITLE
IAM.sp contains redundant controls closes #738

### LINKED ISSUE #738: IAM.sp contains redundant controls
IAM.sp contains the following controls related to password policies:

```
    control.iam_account_password_policy_min_length_14,
    control.iam_account_password_policy_one_lowercase_letter,
    control.iam_account_password_policy_one_number,
    control.iam_account_password_policy_one_symbol,
    control.iam_account_password_policy_one_uppercase_letter,
    control.iam_account_password_policy_reuse_24,
    control.iam_account_password_policy_strong,
    control.iam_account_password_policy_strong_min_length_8,
    control.iam_account_password_policy_strong_min_reuse_24,
```

The last 3 (strong policy controls) are supersets of the granular control checks listed above them.  When run, this causes ambiguous and confusing output due to the mix of granular and summarized checks (and varying minimum length checks, as an example).  I see no reason to leave the last three controls in place as they seem to have been superseded by the granular controls.

### PR DESCRIPTION
**Pull request by khushboo9024**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@khushboo9024 Please see review comments, thanks!

File: conformance_pack/iam.sp [hunk 1]
```diff
@@ -432,19 +432,6 @@ control "iam_account_password_policy_reuse_24" {
   })
 }
 
-control "iam_account_password_policy_strong" {
```
[the remark this answers is not in the document]
Comment (khushboo9024) [reply conformance_pack/iam.sp:435]: May be config rule default values changed, we replaced this control for all our audit manager based benchmarks but we missed to update  for GDPR, FFIEC and cisa_cyber_essentials(non audit manager benchmarks) as these benchmarks are not checked in our monthly check.

[the remark this answers is not in the document]
Comment (khushboo9024) [reply conformance_pack/iam.sp:435]: We are using `iam_account_password_policy_strong_min_reuse_24` as the replacement control. This control matches exactly with aws config rule [iam-password-policy](https://docs.aws.amazon.com/config/latest/developerguide/iam-password-policy.html).

File: conformance_pack/iam.sp [hunk 2]
```diff
@@ -722,7 +709,16 @@ query "iam_account_password_policy_strong_min_reuse_24" {
           and require_symbols = 'true'
           and max_password_age <= 90
         then 'Strong password policies configured.'
-        else 'Strong password policies not configured.'
+        else 'Password policy ' ||
```
[the remark this answers is not in the document]
Comment (khushboo9024) [reply conformance_pack/iam.sp:712]: I've updated the reason to make it clearer and help avoid any misunderstanding for the user
And also this message is updated for all password policy controls, namely `iam_account_password_policy_strong_min_reuse_24` and `iam_account_password_policy_strong_min_length_8`

[the remark this answers is not in the document]
Comment (cbruno10) [reply conformance_pack/iam.sp:715]: @khushboo9024 Can you please share a few screenshots of what the reasons look like when you run the control and it goes to alarm?

File: conformance_pack/iam.sp (hunk 1 above)
[the remark this answers is not in the document]
Comment (cbruno10) [reply conformance_pack/iam.sp:435]: @khushboo9024 Does that reuse_24 control match the exact checks this removed control also had?

File: conformance_pack/iam.sp (hunk 2 above)
[the remark this answers is not in the document]
Comment (khushboo9024) [reply conformance_pack/iam.sp:715]: image

File: conformance_pack/iam.sp (hunk 1 above)
[the remark this answers is not in the document]
Comment (khushboo9024) [reply conformance_pack/iam.sp:435]: `iam_account_password_policy_strong_min_reuse_24` control is a superset of removed control(iam_account_password_policy_strong). Removed control was getting used as config([iam-password-policy](https://docs.aws.amazon.com/config/latest/developerguide/iam-password-policy.html) rule control which was wrong, we have not replaced is with `iam_account_password_policy_strong_min_reuse_24` control.

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**